src/Components/Facility/ConsultationDoctorNotes/index.tsx

```diff
@@ -0,0 +1,131 @@
+import { useState, useEffect } from "react";
+import { useDispatch } from "react-redux";
```

Comment:
You are not supposed to use `useDispatch` in newer code. Use useQuery / request instead